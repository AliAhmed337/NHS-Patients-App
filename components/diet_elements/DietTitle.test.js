import React from 'react';
import { shallow } from 'enzyme';
import { DietTitle } from './index';

describe('Testing diet title component', () => {

	it('renders correcly', () => {
		const wrapper = shallow(
			<DietTitle title="title" />
			);
		wrapper.setProps({ title: 'title' });
		expect(wrapper).toMatchSnapshot();

	});	

});
