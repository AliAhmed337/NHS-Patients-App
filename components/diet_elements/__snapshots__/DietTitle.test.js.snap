// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing diet title component renders correcly 1`] = `ShallowWrapper {}`;
